src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx

```diff
@@ -4,115 +4,33 @@ import { Callout } from "/src/components/callout";
 
-- Deploy a remote canonical interchain token for a pre-existing token on remote chains using the [`deployRemoteCanonicalInterchainToken`](https://github.com/axelarnetwork/interchain-token-service/blob/9edc4318ac1c17231e65886eea72c0f55469d7e5/contracts/InterchainTokenFactory.sol#L257) method on the `InterchainTokenFactory` for each destination chain. This will create the token on each destination chain and its registered under [Mint/Burn](https://docs.axelar.dev/dev/glossary#mintburn) - [Token Manager](https://docs.axelar.dev/dev/glossary#token-manager). They can be deployed to remote chains by anyone and don't depend on a deployer address/salt
+There are two ways to register a canonical token:
 
-Here is an example of deploying a remote canonical token:
-
-```solidity
-/**
- * @notice Deploys a canonical interchain token on a remote chain.
- * @param originalChain The name of the chain where the token originally exists.
- * @param originalTokenAddress The address of the original token on the original
- * chain.
- * @param destinationChain The name of the chain where the token will be deployed.
- * @param gasValue The gas amount to be sent for deployment.
- * @return tokenId The tokenId corresponding to the deployed InterchainToken.
- */
-function deployRemoteCanonicalInterchainToken(
-    string calldata originalChain,
-    address originalTokenAddress,
-    string calldata destinationChain,
-    uint256 gasValue
-) external payable returns (bytes32 tokenId)
-```
-
-- Approve the Interchain Token Service to spend tokens by calling the `approve` method on your token, specifying the Interchain Token Service address and the amount to be approved. Then, proceed to transfer Interchain Tokens between chains via the Interchain Token Service by calling the [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/interfaces/IInterchainTokenService.sol#L210) method
-
-If pre-mint is needed, you must make ERC20 approval to the Interchain Token Factory contract. When tokens are moved from the origin chain to another chain, the token will be locked on the origin chain and minted on the destination chain. If you moved tokens directly from one non-origin chain to another, the token would be burned on the source chain and minted on the destination chain.
+1. Create and register an ERC-20 token on the [Interchain Token Portal](https://testnet.interchain.axelar.dev/).
+1. [Programmatically create a canonical token](/dev/send-tokens/interchain-tokens/developer-guides/programmatically-create-a-canonical-token).
 
 ## Link deployed tokens on multiple chains into Interchain Token(s)
 
-For custom functionality on multiple chains:
```

Comment:
I think you have removed a few important steps they need to do before they Call [deployTokenManager()](https://github.com/axelarnetwork/interchain-token-service/blob/9edc4318ac1c17231e65886eea72c0f55469d7e5/contracts/InterchainTokenService.sol#L276) on the Interchain Token Service to deploy a token manager for the existing version of the token on all chains.